examples/development/nodejs/nodejs-pnpm/devbox.json

```diff
@@ -0,0 +1,12 @@
+{
+    "packages": [
+        "nodejs@18",
+        "nodePackages.pnpm@latest"
+    ],
+    "shell": {
+        "init_hook": [
+            "pnpm install",
+            "pnpm run start"
+        ]
+    }
```

Comment:
would you mind adding a `run_test` devbox script?
That way, our `testscripts` framework will run it to ensure we don't regress this functionality.

(optional) In that, you could verify some output if you want, but even just ensuring it "runs" is a good start.